src/constants/gbhw.asm

```diff
@@ -52,22 +52,30 @@ J_BIT_SELECT EQU 6
 J_BIT_START  EQU 7
 
 ; OAM attribute flags
-OAMF_PRI        EQU %10000000 ; Priority
-OAMF_YFLIP      EQU %01000000 ; Y flip
-OAMF_XFLIP      EQU %00100000 ; X flip
-OAMF_NO_FLIP    EQU %00000000 ; no flip
-OAMF_PAL0       EQU %00000000 ; Palette number; 0,1 (DMG)
-OAMF_PAL1       EQU %00010000 ; Palette number; 0,1 (DMG)
-OAMF_BANK0      EQU %00000000 ; Bank number; 0,1 (GBC)
-OAMF_BANK1      EQU %00001000 ; Bank number; 0,1 (GBC)
+OAM_PRIORITY      EQU 1 << 7    ; Priority (draw background / window over object !important this is only considered after normal priority is resolved!)
+OAM_Y_FLIP        EQU 1 << 6    ; Y flip
+OAM_X_FLIP        EQU 1 << 5    ; X flip
+OAM_NO_FLIP       EQU 0         ; no flip
+OAM_BANK_0        EQU 0 << 3    ; Palette Bank 0 for GBC only
+OAM_BANK_1        EQU 1 << 3    ; Palette Bank 1 for GBC only
+OAM_GB_PAL_0      EQU 0 << 4    ; Palette 0 for GB only
+OAM_GB_PAL_1      EQU 1 << 4    ; Palette 1 for GB only
```

Comment:
I think `GB` to refer specifically to the black-and-white Game Boy is ambiguous ('Game Boy' can be the name of the whole platform as well).

How about `OAM_DMG_PAL_0` and `OAM_DMG_PAL_1` ?

Reply:
In this case I would add something to the comment to explain. My main goal was it, to understand, what is going on, without having to read the GB hardware docs.